### PR TITLE
add cards

### PR DESCRIPTION
**Pull request by Dianakove32**

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1,4 +1,17 @@
 .App {
```
Comment (Fancy322vav) [inline src/App.scss:1]: Please don't use `scss` files here, as you have Material-UI library and all the classes should be declared inside of it. It's redundant here to have so many style libraries.

File: src/App.tsx [hunk 2]
```diff
@@ -1,11 +1,58 @@
-import React from 'react';
+import { Card, CardContent, makeStyles, Typography } from '@material-ui/core';
 import './App.scss';
 
-function App() {
+interface dataTS {
```
Comment (Fancy322vav) [inline src/App.tsx]: Love the interfaces! That's really cool. The only thing, please add an id here to pass it to the `key` in the array. You can add id in a simple way by hardcoding it, or you can do it in a more proficient way, using a library like a uuid for example.

File: src/App.tsx [hunk 3]
```diff
@@ -1,11 +1,58 @@
...
+
   return (
-    <div className="App">
-    </div>
-  );
+      <div className = 'App'>
+      <h1>itechart-react-course</h1>
+          <div className = 'cards-container'>
+          {
+              data.map((el, i) =>
```
Comment (Fancy322vav) [inline src/App.tsx]: Please use id here instead of indexes. Using indexes can lead to a mess while manipulating an array.

File: src/App.tsx (hunk 2 above)
Comment (Fancy322vav) [reply src/App.tsx]: Also it will be better (by pattern of developing) to name it `CardsDataDTO.`

File: src/App.tsx [hunk 4]
```diff
@@ -1,11 +1,58 @@
...
+      title: 'Egypt',
+      description: 'officially the Arab Republic of Egypt, is a transcontinental country spanning the northeast corner of Africa and southwest corner of Asia by a land bridge formed by the Sinai Peninsula.',
+  },
+  {
+      title: 'France',
+      description: 'is a transcontinental country spanning Western Europe and several overseas regions and territories.',
+  },
+]
+
+const useStyles = makeStyles({
```
Comment (Fancy322vav) [inline src/App.tsx]: You can add a `theme` argument here and use it while setting any sizes or colors.

**Review by Fancy322vav — APPROVED** [review]
Looks really awesome! 👍  Left some comments with slight requested changes. They can be done in the PR to the task3, as some of them are related right to this task :)

File: src/App.tsx (hunk 2 above)
Comment (Dianakove32) [reply src/App.tsx]: done

File: src/App.tsx (hunk 3 above)
Comment (Dianakove32) [reply src/App.tsx]: done

File: src/App.scss (hunk 1 above)
Comment (Dianakove32) [reply src/App.scss:1]: done

File: src/App.tsx (hunk 4 above)
Comment (Dianakove32) [reply src/App.tsx]: remove Material-UI library

**Pull request merged**